layouts/shortcodes/studio-hub.html

```
@@ -0,0 +1,259 @@
+    {{ else }}
+    <header class="authz-page-head">
+        <div class="authz-wrap">
+            <div class="authz-crumbs">
+                <a href="{{ "altinn-studio/v9/" | relLangURL }}">Altinn Studio</a> / {{ .Page.Title }}
+            </div>
+            <div class="authz-eyebrow">Altinn Studio</div>
+            <h1>{{ .Page.Title }}</h1>
+            <p>{{ .Page.Description }}</p>
+        </div>
+    </header>
+    <section class="authz-section authz-wrap{{ if eq $kind " getting-started" }} authz-task-overview{{ end }}">
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ < ].

(spec-char-escape)